src/components/drawer/drawer.tsx

```diff
@@ -1,51 +1,63 @@
-import { ChevronRightIcon } from '@radix-ui/react-icons';
-import classnames from 'classnames';
 import { useEffect } from 'react';
+import { createPortal } from 'react-dom';
+import { ClientOnly } from 'remix-utils/client-only';
+import classNames from 'classnames';
+import { AnimatePresence, motion } from 'framer-motion';
+import { RemoveScroll } from 'react-remove-scroll';
 import styles from './drawer.module.scss';
 
-export interface DrawerProps {
-    className?: string;
+interface DrawerProps {
+    open: boolean;
```

Comment:
why did you change this prop name ?

Reply:
it's copied from the Drawer of the ReClaim template